backend/typescript/middlewares/validators/campValidators.ts

```diff
@@ -75,30 +75,50 @@ export const createCampDtoValidator = async (
   if (!validatePrimitive(body.description, "string")) {
     return res.status(400).send(getApiValidationError("description", "string"));
   }
-  if (!validatePrimitive(body.earlyDropoff, "string")) {
+  if (body.earlyDropoff && !validatePrimitive(body.earlyDropoff, "string")) {
```

Comment:
I'm pretty sure `validatePrimitive` handles the null/undefined checks?

Reply:
Added the existance check because earlyDropoff is not a required field. if we only use the validatePrimitive then it would always fail for camps that don't offer EDLP since it won't be a type string